.travis.yml

```diff
@@ -10,4 +10,4 @@ install:
   - gem install awesome_bot
 
 script:
-  - awesome_bot README.md --allow-redirect --white-list "www.0day.today,mvfjfugdwgc5uwho.onion,creativecommons.org,netsparker.com,www.shodan.io,www.mhprofessional.com,ghostproject.fr,www.zoomeye.org"
+  - awesome_bot README.md --allow-redirect --white-list "www.defcon.org,www.0day.today,mvfjfugdwgc5uwho.onion,creativecommons.org,netsparker.com,www.shodan.io,www.mhprofessional.com,ghostproject.fr,www.zoomeye.org"
```

Comment:
Was the issue with DEF CON's website [this DNS resolution error on Travis](https://travis-ci.org/enaqx/awesome-pentest/builds/512129851#L1135)? If so, I think that's an intermittent issue. I'm not having any trouble getting DNS resolution for their domain. I think it's probably safe to drop this domain from the white list, and the fewer white listed domains, the better.